<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>939</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>939</width>
    <height>939</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>939</width>
    <height>939</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FacultyFinder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>671</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#ffffff;&quot;&gt;Faculty Finder&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="teacher_button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>600</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(164, 255, 121)</string>
    </property>
    <property name="text">
     <string>Faculty</string>
    </property>
   </widget>
   <widget class="QPushButton" name="student_button">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>600</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(164, 255, 121)</string>
    </property>
    <property name="text">
     <string>Student</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res2.qrc"/>
 </resources>
 <connections/>
</ui>
